src/index.html

```diff
@@ -11,7 +11,188 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset
+        name="personal-info"
+        class="fieldset"
+      >
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            name="surname"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            autocomplete="off"
+            name="name"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            required
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="birth-date"
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset
+        name="registration"
+        class="fieldset"
+      >
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="6"
+            maxlength="16"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset
+        name="interstingFacts"
+        class="fieldset"
+      >
+        <legend>An interesting fact about you!</legend>
+
+        Do you love cats?
+        <input
+        type="radio"
+        name="cat"
+        value="yes"
+        required
+        >
+        <label class="radiobutton">
+          Yes
+        </label>
```

Comment:
```suggestion
        <input
          type="radio"
          name="cat"
          value="yes"
          required
        >
        <label class="radiobutton">
          Yes
        </label>
```
You can also put input inside label too, so user can click on yes/no and trigger the input